packages/alice/lib/core/alice_core.dart

```diff
@@ -116,56 +90,61 @@ class AliceCore {
     }
     if (!_isInspectorOpened) {
       _isInspectorOpened = true;
-      AliceNavigation.navigateToCallsList(core: this, logger: _aliceLogger)
+      AliceNavigation.navigateToCallsList(core: this)
```

Comment:
This should be re-written using `async`/`await` as it's waaay clearer.

Reply:
Ok fixed.